sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_manage_models.py

```diff
@@ -97,6 +97,10 @@ def sample_manage_models():
     print(f"Description: {my_model.description}")
     print(f"Model created on: {my_model.created_date_time}")
     print(f"Model expires on: {my_model.expiration_date_time}")
+    if my_model.warnings:
```

Comment:
Just curious, if this was manually added or generated?

Reply:
Added manually.